### ISSUE TITLE
AppImage does not work on Ubuntu 18.04.1 LTS

### ISSUE THREAD

**Issue by matu3ba** [RobertKrajewski/Sync-my-L2P#127]
Since recent time the appimage does not work
`$ ./SyncMyL2P-2.3.0-linux.AppImage`
> Speicherzugriffsfehler (Speicherabzug geschrieben)

gdb stack returns

> Thread 1 "AppRun" received signal SIGSEGV, Segmentation fault.
> 0x00000000000011e0 in ?? ()

backtrace returns

> #0  0x00000000000011e0 in ?? ()
> #1  0x00007ffff7de5733 in call_init (env=0x6fc990, argv=0x7fffffffdfb8, argc=1, l=<optimized out>) at dl-init.c:72
> #2   _dl_init (main_map=main_map@entry=0x702a80, argc=1, argv=0x7fffffffdfb8, env=0x6fc990) at dl-init.c:119
> #3  0x00007ffff7dea1ff in dl_open_worker (a=a@entry=0x7fffffffd790) at dl-open.c:522
> #4  0x00007ffff64132df in __GI__dl_catch_exception (exception=0x7fffffffd770, operate=0x7ffff7de9dc0 <dl_open_worker>, args=0x7fffffffd790) at dl-error-skeleton.c:196
> #5  0x00007ffff7de97ca in _dl_open (file=0x6f7308 "libgnomevfs-2.so.0", mode=-2147483647, caller_dlopen=0x7ffff6ebf8ff, nsid=<optimized out>, argc=1, 
>     argv=<optimized out>, env=0x6fc990) at dl-open.c:605
> #6  0x00007ffff3c1ef96 in dlopen_doit (a=a@entry=0x7fffffffd9c0) at dlopen.c:66
> #7  0x00007ffff64132df in __GI__dl_catch_exception (exception=exception@entry=0x7fffffffd960, operate=0x7ffff3c1ef40 <dlopen_doit>, args=0x7fffffffd9c0)
>     at dl-error-skeleton.c:196
> #8  0x00007ffff641336f in __GI__dl_catch_error (objname=0x6a4270, errstring=0x6a4278, mallocedp=0x6a4268, operate=<optimized out>, args=<optimized out>)
>     at dl-error-skeleton.c:215
> #9  0x00007ffff3c1f735 in _dlerror_run (operate=operate@entry=0x7ffff3c1ef40 <dlopen_doit>, args=args@entry=0x7fffffffd9c0) at dlerror.c:162
> #10 0x00007ffff3c1f051 in __dlopen (file=<optimized out>, mode=<optimized out>) at dlopen.c:87
> #11 0x00007ffff6ebf8ff in ?? () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Core.so.5
> #12 0x00007ffff6eb8ff8 in QLibrary::load() () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Core.so.5
> #13 0x00007ffff6eb9528 in QLibrary::resolve(char const*) () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Core.so.5
> #14 0x00007ffff6ebbf76 in QLibrary::resolve(QString const&, int, char const*) () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Core.so.5
> #15 0x00007ffff792170b in ?? () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #16 0x00007ffff791e58a in ?? () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #17 0x00007ffff78a44dd in QStyleFactory::create(QString const&) () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #18 0x00007ffff783d755 in QApplication::style() () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #19 0x00007ffff783db1d in QApplicationPrivate::initialize() () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #20 0x00007ffff783db6f in QApplicationPrivate::construct() () from /tmp/.mount_SyncMyHlV00Y/usr/bin/../lib/libQt5Widgets.so.5
> #21 0x0000000000412d0b in main ()

Any idea for the reason?

**Comment by RobertKrajewski** [comment]
So it worked till last week? Then it's because at the beginning of the semester changes were probably made to the L2P.

**Comment by Bergiu** [comment]
Maybe this is the same problem like #122 and #128. Does it work with "Media Library" disabled on the options tab?

**Comment by matu3ba** [comment]
Here a very simple reproducing script. I dont see any "Media Library" flag. 
Also I dont see any flag in qmake or make providing the keyword "media".
```
#!/bin/bash  
#failure reproducing  
wget https://github.com/RobertKrajewski/Sync-my-L2P/releases/download/v2.3.0/SyncMyL2P-2.3.0-linux.AppImage  
chmod +x SyncMyL2P-2.3.0-linux.AppImage  
time ./SyncMyL2P-2.3.0-linux.AppImage  
```

**Comment by RobertKrajewski** [comment]
Have you been using Sync-my-L2P with activated auto-login? If yes, I assume, that you see the program start before it crashes. You have to deactivate the "Download Media Library" files in the options tab in the program.

**Comment by matu3ba** [comment]
Got somehow fixed in version 2.4
